ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.15-periodics.yaml

```diff
@@ -0,0 +1,83 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 0 * * 2,5
+  decorate: true
+  extra_refs:
+  - base_ref: community-4.15
+    org: openshift
+    repo: windows-machine-config-operator
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: okd
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-windows-machine-config-operator-community-4.15-okd-aws-e2e-operator
+  reporter_config:
+    slack:
+      channel: '#forum-winc'
```

Comment:
#forum-ocp-winc
And that reminds me that it should have been changed for the other branches too